front/app/modules/commercial/customizable_navbar/admin/containers/EditPageForm/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Decides with of the two above forms should render based on whether the page is in the navbar or not.